images/jupyter-user/bundle.sh

```diff
@@ -39,3 +39,14 @@ touch ${ARCHIVE_DIR}/pip.conf
 
 cd ${DIR}
 tar czvf aws-orbit_jupyter-user.tar.gz ./aws-orbit_jupyter-user
+
+
+# Cloning forked jupyterlab databrew extension
+git clone https://github.com/stthoom/aws-glue-databrew-jupyter-extension.git
+cd aws-glue-databrew-jupyter-extension
+git branch
+git checkout feature/orbit_databrew_extension
+cd ${DIR}
+tar czvf aws-glue-databrew-jupyter-extension.tar.gz ./aws-glue-databrew-jupyter-extension
```

Comment:
why do we bundle this as a .tar.gz into the image and not install it directly?

Reply:
Thought of avoiding cloning and branch checkout steps(layers) in docker file. Will be replaced with "pip install " once we get JL 3 compatible databrew extension with pip packaging capability. As of now it is JL2 compatible and npm based. We forked and installing directly from source with JL version changes.